.github/workflows/try.yml

```diff
@@ -20,13 +24,15 @@ jobs:
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
-      #- uses: actions/checkout@master
-      #  with:
-      #    name: qiboteam/qibo 
-      - run: git pull
-      - run: git clone https://github.com/qiboteam/qibo.git  
-      - run: ls -l
+
+      - name: Clone qibo repository
+        uses: actions/checkout@v2
+        with:
+          repository: qiboteam/qibo
+          path: "~/"
```

Comment:
It should be already the default, so drop for the time being.

Reply:
Eventually, you'll want to do it for multiple repos, but we will do in separate jobs, since they don't need each other.